packages/sku/src/services/telemetry/index.ts

```diff
@@ -3,7 +3,7 @@ import os from 'node:os';
 import { requireFromCwd } from '@/utils/cwd.js';
 import isCI from '@/utils/isCI.js';
 import provider from './provider.js';
-import skuPackageJson from '../../../package.json' with { type: 'json' };
+import skuPackageJson from 'sku/package.json' with { type: 'json' };
```

Comment:
`sku/package.json` is a more stable reference to the `package.json`, so if this file moves it won't need to be updated.